### ISSUE TITLE
Change time of System Sem Conv Stability WG

### ISSUE THREAD

**Issue by ChrsMark** [open-telemetry/community#2064]
ref: https://docs.google.com/document/d/1p5TH57t43XpxA48onLzX4PIr3g6ydYKCtR_AUlsCnQk/edit

As per our discussions at the [WG's slack channel](https://cloud-native.slack.com/archives/C05CTFE9U4A/p1713340198438569), we would like to change the time of the weekly meeting in order to resolve the time conflict with the Entities and Profiling WGs.

The new agreed time is `2:30PM UTC` on the same day (`Thursday`).

### TODOs

1) update calendar event
2) update zoom link (if needed)

cc: @mx-psi , @dmitryax @braydonk @jsuereth

**Comment by mx-psi** [comment]
I support. @ChrsMark I think since we already discussed this on the Slack channel, we can file a PR to change the time [here](https://github.com/open-telemetry/community?tab=readme-ov-file#specification-sigs), and let Josh handle the actual change on Google Calendar. Wdyt?

**Comment by ChrsMark** [comment]
Yeap, I can file a PR and let's make sure we will find someone to change the Calendar as well.

**Comment by svrnm** [comment]
@ChrsMark please update the time in

https://github.com/open-telemetry/community/blob/main/sigs.yml

and afterwards run `make table-generation` locally to update README.md

**Comment by ChrsMark** [comment]
I have opened the PR: https://github.com/open-telemetry/community/pull/2065

**Comment by svrnm** [comment]
The PR is merged, but I guess the calendar event still needs to be updated?

**Comment by ChrsMark** [comment]
That's correct @svrnm . Someone with permissions will have to do it for us :)

**Comment by svrnm** [comment]
> That's correct @svrnm . Someone with permissions will have to do it for us :)

let me give it a try

**Comment by svrnm** [comment]
I moved it, it is also already reflected in my calendar, let me know if it worked

**Comment by ChrsMark** [comment]
I received the notification. It worked for me as well! Thank's a lot!

**Comment by arminru** [comment]
Thanks @svrnm!
